<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LootTablePool</class>
 <widget class="QTabWidget" name="LootTablePool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="rollsTab">
   <attribute name="title">
    <string>Rolls</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>The number of times the loot table will pick an item from its pool.</string>
      </property>
      <property name="text">
       <string>Rolls:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="NumericInput" name="rollsInput">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="bonusRollsLabel">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Bonus rolls from luck.</string>
      </property>
      <property name="text">
       <string>Bonus rolls:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="NumericInput" name="bonusRollsInput">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="entriesTab">
   <attribute name="title">
    <string>Entries (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="entriesInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="functionsTab">
   <attribute name="title">
    <string>Functions (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="functionsInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="conditionsTab">
   <attribute name="title">
    <string>Conditions (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="conditionsInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataWidgetInterface</class>
   <extends>QFrame</extends>
   <header location="global">datawidgetinterface.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NumericInput</class>
   <extends>QFrame</extends>
   <header location="global">numericinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
